lib/daru/dataframe.rb

```diff
@@ -1902,14 +1902,13 @@ def transpose
 
     # Pretty print in a nice table format for the command line (irb/pry/iruby)
     def inspect spacing=10, threshold=15
-      row_headers = index.is_a?(MultiIndex) ? index.sparse_tuples : index.to_a
```

Comment:
I was getting `Assignment Branch Condition size` error in rubocop. so I added 2 private methods to get `headers` and `row_headers`.